.github/workflows/rpm-build.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       - name: Build rpmbuild Container
         uses: docker/build-push-action@v2
         with:
-          context: https://github.com/riege/rpm-build.git#main
+          context: https://github.com/riege/rpm-build.git#rhel6
```

Comment:
Note that this change is **temporary** because otherwise the workflow would not find the RHEL 6 Dockerfile.